algorithms/kernel/dtrees/dtrees_model_impl_common.h

```diff
@@ -68,6 +68,7 @@ void nodeToTable(const NodeBase& node, size_t iRow, size_t& iCur, DecisionTreeNo
             }
         }
         row.featureIndex = -1;
+        row.featureValueOrResponse = 0;
```

Comment:
in regression case we have:
void writeLeaf(const TLeaf& l, DecisionTreeNode& row)
{
    row.featureValueOrResponse = l.response;
}
This field `featureValueOrResponse` is not  initialized only for classification flow.
Seems we have to add the same for regression flow: `row.leftIndexOrClass = 0`